### PR TITLE
Newsletter 13: Akigi

### PR DESCRIPTION
**Pull request by ozkriff**
@chinedufn r? Should something be changed or added? :)

_Part of #232  ("Newsletter 13")_

### CODE REVIEW

**Comment by ozkriff** [comment]
:timer_clock: Merging in a few hrs if there're no objections.

File: content/posts/newsletter-013/index.md [hunk 1]
```diff
@@ -301,6 +301,29 @@ Jazarro on [the Amethyst discord][amethyst-discord].
 [kv-ii]: https://en.wikipedia.org/wiki/King%27s_Valley_II
 [amethyst-discord]: https://discord.com/invite/amethyst
 
+### [Akigi][akigi]
+
+[Akigi][akigi] is a magical multiplayer online world where humans aren't the
```
Comment (chinedufn) [inline content/posts/newsletter-013/index.md:306]: This actually isn't the tag line / theme of the game anymore - although to be fair it's been changing every few months ...

Lately it's just been `The multiplayer online world of Akigi.`

**Review by chinedufn — APPROVED** [review]
Looks awesome, thank you!!!

File: content/posts/newsletter-013/index.md (hunk 1 above)
Comment (ozkriff) [reply content/posts/newsletter-013/index.md]: Updated!

**Review by 17cupsofcoffee — APPROVED** [review]

**Pull request merged**